snippets/ztp-performance-profile.adoc

```diff
@@ -8,9 +8,8 @@ metadata:
   name: openshift-node-performance-profile <1>
 spec:
   additionalKernelArgs:
-    - idle=poll
-    - rcupdate.rcu_normal_after_boot=0
-    - "efi=runtime" <2>
```

Comment:
You removed the <2> parameter, but didn't renumber the following.

Reply:
well spotted :)